manifests/supervisorcluster/1.29/cns-csi.yaml

```diff
@@ -102,6 +102,9 @@ rules:
   - apiGroups: ["cns.vmware.com"]
     resources: ["cnsvolumeinfoes"]
     verbs: ["create", "get", "list", "watch", "delete", "patch"]
+  - apiGroups: ["crd.nsx.vmware.com"]
+    resources: ["networkinfos"]
+    verbs: ["get", "watch", "list"]
```

Comment:
@gohilankit has added 1.30 cns-csi yaml file. Can you make change in that file as well?

Reply:
Rebased and added.